eng/common/scripts/Verify-Links.ps1

```diff
@@ -262,7 +262,13 @@ function GetLinks([System.Uri]$pageUri)
 
   return $links;
 }
+Write-Host "There are $($urls.length) files."
```

Comment:
I assume this is just for testing?

Reply:
Yes, it is for testing. Will remove.
I found the script did not pick up HEAD correctly. I have 2 readme changes, but only one got picked up.
```
git diff origin/master HEAD --name-only -- *.md
```
I am trying to use commit and test

```
git diff origin/master "${env:SYSTEM_PULLREQUEST_SOURCECOMMITID}" --name-only -- *.md
```